huntsman/tests/test_huntsman.py

```diff
@@ -249,10 +249,12 @@ def test_pyro_camera(config, camera_server):
     assert obs.cameras['camera.simulator.001'].is_connected
 
 
+@pytest.mark.skip
 def test_run_wait_until_safe(observatory, cmd_publisher, msg_subscriber):
     os.environ['POCSTIME'] = '2016-09-09 10:00:00'
 
-    observatory.db.insert_current('weather', {'safe': False})
+    # Make sure DB is clear for current weather
+    observatory.db.clear_current('weather')
```

Comment:
Did this change actually do anything or was it just the attempt to fix this test?

Reply:
Just an attempt to fix the test. The version of this test in POCS does this, so I copied it to `huntsman-pocs` to see if it made a difference. It did, but the difference didn't fix the test.